src/components/projectStatus/ProjectTableContainer.tsx

```diff
@@ -158,9 +171,25 @@ const ProjectTableContainer: React.FC<Props> = props => {
             <>
               {project.categories.map((category: any) => (
                 <>
-                  <Title level={5} key={category.id}>
-                    {category.name}
-                  </Title>
+                  <Flex>
+                    <Title level={5} key={category.id}>
+                      {category.name} 
+                    </Title>
+                    <Text pl={2} pt={1} pb={0} fontSize="xs" color="blue.500" _hover={{ cursor: "pointer", textDecoration: "underline" }} onClick={() => handleComponentClick(project.id, category.name)}>
+                      What is this?
```

Comment:
Same for the rest of the "What is this" 
<img width="360" alt="Screenshot 2023-07-31 at 1 48 23 PM" src="https://github.com/HackGT/timber/assets/8273038/103b36c0-a3e9-4253-8ab6-c92da697e8dd">